src/main/java/by/iba/party/controller/party/PartyController.java

```diff
@@ -100,25 +108,27 @@ public void delete(@PathVariable Integer id) {
 
     @PostMapping(value = "/add")
     @ResponseStatus(HttpStatus.OK)
-    public Party addNew(@RequestBody Party party) {
-        partyService.save(party);
-        return party;
+    public PartyDto addNew(@RequestBody PartyDto partyDto) {
+        return partyService.save(partyDto);
     }
 
     @PostMapping(value = "/{party_id}/add/user/{user_id}")
-    public void addUserToParty(@PathVariable(value = "party_id") Party party, @PathVariable(value = "user_id") User userInfo) {
-        partyService.addUserToParty(party, userInfo);
-    }
-
-    @GetMapping(value = "/{party_id}/check/user/{user_id}")
-    public User checkUserToParty(@PathVariable(value = "party_id") Party party, @PathVariable(value = "user_id") User user) {
-        if (partyService.checkUserToParty(party, user)) return user;
-        else return null;
+    public void addUserToParty(@PathVariable(value = "party_id") PartyDto partyDto, @PathVariable(value = "user_id") UserDto userInfoDto) {
+        partyService.addUserToParty(partyDto, userInfoDto);
     }
 
     @GetMapping(value = "/{party_id}/users")
-    public List<User> getAllUsersOnTHisParty(@PathVariable(value = "party_id") Party party) {
-        return partyService.findById(party.getId()).get().getUsers();
+    public List<UserDto> getAllUsersOnTHisParty(@PathVariable(value = "party_id") Integer id) throws NoEntityException {
+        return partyService.findById(id).getUsers();
     }
 
+    private ProductDto findProductById(Integer id) {
+        try {
+            return productService.findById(id);
+        }
+        catch (NoEntityException e) {
+            log.debug("No product with such id");
```

Comment:
Please put more details to message "", add e to log as 
`log.debug("No product with ID: {} exists", id, e);`